src/commands/releases.rs

```diff
@@ -588,7 +592,14 @@ fn execute_set_commits<'a>(
         let commits = generate_patch_set(&repo, commit_log, prev_commit, &parsed)?;
 
         if commits.is_empty() {
-            bail!("No commits found. Leaving release alone.");
+            // NOTE: Make it a default behavior on next major release instead?
```

Comment:
Maybe we explicitly say `TODO(v2):` so it's easier to grep? Then we can make a decision on what we wanna do when we think about the next major bump.

Reply:
Updated

Reply:
I don't know if that is a good idea. A release could consist of configuration changes through your environment. In those cases, I still want to have a release on Sentry. This could be a very common use case for every one using the 12 factor app approach.

Reply:
Noted. Thanks for the feedback!